pkg/models/user.go

```diff
@@ -215,6 +215,7 @@ type UserProfileDTO struct {
 	OrgId          int64  `json:"orgId"`
 	IsGrafanaAdmin bool   `json:"isGrafanaAdmin"`
 	IsDisabled     bool   `json:"isDisabled"`
+	AuthModule     string `json:"authModule"`
```

Comment:
AuthModule will have to be a list of strings as a user can be linked with more than one module.